src/screens/SendSheet.js

```diff
@@ -702,7 +707,8 @@ export default function SendSheet(props) {
       isEmpty(gasFeeParamsBySpeed) ||
       !selectedGasFee ||
       isEmpty(selectedGasFee?.gasFee) ||
-      !toAddress
+      !toAddress ||
+      (currentNetwork === Network.optimism && l1GasFeeOptimism === null)
```

Comment:
kinda confused why this wasn't already here, am i missing something?

Reply:
thanks for pointing this out... this fixes a bug that isn't a problem for sending optimism erc20s. makes me think it shouldnt have to be a problem for nfts either, so gonna investigate that

Reply:
This still pending?